src/sagemaker/serve/builder/transformers_builder.py

```diff
@@ -301,6 +348,27 @@ def _build_for_transformers(self):
         self.secret_key = None
         self.model_server = ModelServer.MMS
 
+        if self.inference_spec:
+
+            if not os.path.exists(self.model_path):
```

Comment:
nit: `os.makedirs(self.model_path, exist_ok=True)`

Reply:
I'll make this edit, thank you